src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin-top: 10px;
+}
+
+.form-fieldset {
+  margin-bottom: 20px;
+}
+
+legend + .form-field {
```

Comment:
1. use class as css selector instead of tag name
2. you can use :not(:first-child) to define styless for all elements except the first child